lib/main.gi

```diff
@@ -1392,7 +1393,20 @@ else
                      )
                 ];        
     fi;       
-
+  return GlobalSplittingOfCyclotomicAlgebra(A);
```

Comment:
Part of this commit above went into PR #65, except the commented out code below, which has been uncommented and modified there.